integrationtest/fullstack/library/src/test/java/org/hibernate/search/integrationtest/fullstack/support/JtaEnvironment.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.fullstack.support;
+
+import org.junit.rules.ExternalResource;
+
+import com.arjuna.ats.jta.utils.JNDIManager;
+import org.jnp.server.NamingBeanImpl;
+
+public class JtaEnvironment extends ExternalResource {
```

Comment:
Do we really need JTA in our case? I would have thought it was only required if we needed XA transactions.

Reply:
If I remember well JTS is the JTA support for XA, JTA would be used even for local transaction. I'll check what happens with out it.